.github/workflows/build/conf/topologies/knoxtoken.xml

```diff
@@ -0,0 +1,56 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<topology>
+        <gateway>
+			<provider>
+			            <role>federation</role>
+			            <name>JWTProvider</name>
+			            <enabled>true</enabled>
+			            <param>
+			                <name>knox.token.use.cookie</name>
+			                <value>false</value>
+			            </param>
+			            <param>
+			                <name>jwt.expected.sigalg</name>
+			                <value>RS256</value>
+			            </param>
+			        </provider>
+        </gateway>
+        <service>
+            <role>KNOXTOKEN</role>
+            <param>
+                <name>knoxsso.token.ttl</name>
+                <value>86400000</value>
+            </param>
+        </service>
+		<service>
+        <role>KNOX-AUTH-SERVICE</role>
+        <param>
+            <name>preauth.auth.header.actor.id.name</name>
+            <value>x-knox-actor-username</value>
+        </param>
+        <param>
+            <name>preauth.auth.header.actor.groups.prefix</name>
+            <value>x-knox-actor-groups</value>
+        </param>
+        <param>
+            <name>preauth.group.filter.pattern</name>
+            <value>[^\s]+</value>
+        </param>
+    </service>
+</topology>
```

Comment:
It was hard to read this topology, can you please apply the appropriate level of indentation?
Moreover, I'm not sure I understand the reason why this topology is added. It's not used anywhere...

Reply:
I still don't see where this topology is used.
Based on the JWT federation provider and the `KNOXTOKEN` service I can see some token exchange flows here.

What's the purpose of the `KNOX-AUTH-SERVICE`?

Reply:
I am adding tests as we go, for the first iteration i want to add token related tests that support the istio external authorizer flow, hence the use of `KNOX-AUTH-SERVICE`. I'll follow up with tests that use this topology.